scenarios1/01_An_Orcish_Assault.cfg

```diff
@@ -19,8 +19,12 @@
         profile=portraits/Efraim.png
         canrecruit=yes
         controller=human
-        gold=150
+        {GOLD 150 150 120}
+#ifdef HARD
         recruit=Fencer,Thief,Spearman,Bowman,Heavy Infantryman,Poacher
+#else
+        recruit=Fencer,Thief,Spearman,Bowman,Heavy Infantryman,Poacher,Rogue
```

Comment:
So that the player has to advance their army from low level units.

Reply:
The diff looks like if you had _added_ the Rogue to the recruit list on easier difficulties, not removed it on hard.

Reply:
Yes.  Like I said, recruit lists work dumb.   I could say ifndef HARD I suppose, but in my experience most people think ifdef is easier to follow.

If it's just the diff that seems odd, it's probably because I removed the recall list from Tutorial (better check I remember to git add that one).